hotel_restaurant/static/description/index.html

```diff
@@ -0,0 +1,448 @@
+!! changes will be overwritten.                   !!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! -->
+<p><a class="reference external" href="https://odoo-community.org/page/development-status"><img alt="Beta" src="https://img.shields.io/badge/maturity-Beta-yellow.png" /></a> <a class="reference external" href="http://www.gnu.org/licenses/agpl-3.0-standalone.html"><img alt="License: AGPL-3" src="https://img.shields.io/badge/licence-AGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/vertical-hotel/tree/12.0/hotel_restaurant"><img alt="OCA/vertical-hotel" src="https://img.shields.io/badge/github-OCA%2Fvertical--hotel-lightgray.png?logo=github" /></a> <a class="reference external" href="https://translation.odoo-community.org/projects/vertical-hotel-12-0/vertical-hotel-12-0-hotel_restaurant"><img alt="Translate me on Weblate" src="https://img.shields.io/badge/weblate-Translate%20me-F47D42.png" /></a> <a class="reference external" href="https://runbot.odoo-community.org/runbot/157/12.0"><img alt="Try me on Runbot" src="https://img.shields.io/badge/runbot-Try%20me-875A7B.png" /></a></p>
+<p>This Module is providing table booking facilities and Managing customers orders</p>
+<p>You can manage:</p>
+<ul class="simple">
+<li>Restaurant Configuration (Food Item, Table)</li>
+<li><strong>Table Reservation</strong></li>
+</ul>
+<img alt="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res2.png" src="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res2.png" style="width: 750px;" />
+<ul class="simple">
+<li><strong>Manage Food Orders</strong></li>
+</ul>
+<img alt="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res3.png" src="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res3.png" style="width: 750px;" />
+<ul class="simple">
+<li><strong>Manage Kitchen Order ticket</strong></li>
+</ul>
+<img alt="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res4.png" src="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res4.png" style="width: 750px;" />
+<ul class="simple">
+<li><strong>Quick Table Reservation Facility With Its Food Order</strong></li>
+</ul>
+<img alt="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res5.png" src="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res5.png" style="width: 750px;" />
+<ul class="simple">
+<li><strong>Order’s History In Hotel Folio</strong></li>
+</ul>
+<img alt="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res6.png" src="https://raw.githubusercontent.com/OCA/vertical-hotel/11.0/hotel_restaurant/static/description/res6.png" style="width: 750px;" />
```

Comment:
Make sure about the screenshot we attached, if the layout is different we need to update it.